packages/apps/docs/src/components/Layout/components/SideMenu/SideMenu.tsx

```diff
@@ -25,12 +23,6 @@ export const SideMenu: FC<IProps> = ({ closeMenu, menuItems }) => {
   const { clickSubMenu, treeRef } = useSideMenu(closeMenu, menuItems);
   const router = useRouter();
   const searchRef = useRef<HTMLInputElement>(null);
-  const { isMenuOpen } = useMenu();
-
-  useEffect(() => {
```

Comment:
focus in the sidemenu on the search.
will automatically open the keyboard. you dont want that.

Reply:
yes, it's annoying as soon as open the menu page getting zoomed in